source/data-formats/extended-json.txt

```diff
@@ -0,0 +1,203 @@
+       | This format prioritizes type preservation at the loss of human-readability and
+         interoperability with older formats.
+
+   * - **Relaxed Mode**
+     - | JSON representation that describes BSON documents with some type information loss.
+       | This format prioritizes human-readability and interoperability at the loss of
+         certain type information.
+
+To learn more about JSON, BSON, and Extended JSON, see
+`our article about JSON and BSON <https://www.mongodb.com/resources/basics/json-and-bson>`__
+and :manual:`Extended JSON </reference/mongodb-extended-json/>` in the {+mdb-server+} manual.
+
+.. _extended_json_example_section:
+
+Extended JSON Examples
+~~~~~~~~~~~~~~~~~~~~~~
+
+The following examples show a document containing an ObjectId, date, and long
+number field represented in the Extended JSON format. Click the tab that
+corresponds to the format of the example you want to see:
+
+.. tabs::
+
+   .. tab:: Extended
+      :tabid: extended-format
+
+      .. code-block:: json
+
+         {
+           "_id": { "$oid": "573a1391f29313caabcd9637" },
+           "createdAt": { "$date": { "$numberLong": "1601499609" }},
+           "numViews": { "$numberLong": "36520312" }
+         }
+
+   .. tab:: Relaxed Mode
+      :tabid: relaxed-mode-format
+
+      .. code-block:: json
+
+         {
+           "_id": { "$oid": "573a1391f29313caabcd9637" },
+           "createdAt": { "$date": "2020-09-30T18:22:51.648Z" },
+           "numViews": 36520312
+         }
+
+Write Extended JSON
+-------------------
+
+You can write an Extended JSON string from a BSON document object by using the
+``EJSON.stringify()`` method.
+
+The following example outputs an Extended JSON string in the Relaxed format:
+
+.. io-code-block::
+
+   .. input::
+      :language: js
+
+      const { Binary, Code, EJSON } = require('bson');
+
+      doc = [
+         {"foo": [1, 2]},
+         {"bar": {"hello": "world"}},
+         {"code": new Code("function x() { return 1; }", {})},
+         {"bin": Binary.createFromBase64("AQIDBA==", 0)}
+      ]
+
+      const ejson_str = EJSON.stringify(doc);
+      console.log(ejson_str);
+
+   .. output::
+      :language: none
+      :visible: false
+
+      [{"foo":[1,2]},{"bar":{"hello":"world"}},{"code":{"$scope":{},"$code":"function x() { return 1; }"}},{"bin":{"$binary":{"base64":"AQIDBA==","subType":"00"}}}]
+
+By default, the ``stringify()`` method returns the Extended JSON string in the 
+Relaxed format. To specify the Canonical format, set the ``relaxed`` 
+option to ``false``.
+
+The following example shows how to output Extended JSON in the Canonical format:
+
+.. io-code-block::
+
+   .. input::
+      :language: js
+
+      const { Binary, Code, EJSON } = require('bson');
+
+      doc = [
+         {"foo": [1, 2]},
+         {"bar": {"hello": "world"}},
+         {"code": new Code("function x() { return 1; }", {})},
+         {"bin": Binary.createFromBase64("AQIDBA==", 0)}
```

Comment:
Same mods as above. 

Maybe we should replace the binary example with a Date one? we have the "$numberInt"s showing up here, but the difference between Date relaxed true/false is more significant, it demonstrates how one is readable and the other is intended more for transmission/storage.

Reply:
Replaced with a Date object. Let me know what you think!